common/beerocks/bcl/source/beerocks_logging.cpp

```diff
@@ -506,7 +529,13 @@ void logging::apply_settings()
     syslogConf.set(el::Level::Trace, el::ConfigurationType::Enabled,
                    string_utils::bool_str(m_syslog_levels.trace_enabled()));
 
-    el::Loggers::reconfigureLogger("default", defaultConf);
+    auto logger = el::Loggers::getLogger(m_logger_id);
+    if (!logger) {
+        LOG(ERROR) << "invalid logger!";
+        return;
```

Comment:
If this function can now fail, it should return a (boolean?) value...

Reply:
It could have failed before, I've just moved it.
Anyway, I'm not really sure what to do with that and I don't want to do an additional refactoring as part of this PR.